src/ZODB/_compat.py

```diff
@@ -12,20 +12,20 @@
 #
 ##############################################################################
 import sys
+from six import PY3
 
 IS_JYTHON = sys.platform.startswith('java')
 
-try:
+
+if not PY3:
```

Comment:
The other half of the branch (the "if PY3" case) handles PyPy3. There was already a special case for any Python 3 version (they all used zodbpickle already), but it was being detected implicitly through the failure of `import cPickle`, which never existed on any Python 3 (but did exist on PyPy2, albeit the same as regular `pickle`).